tests/PHPUnit/System/expected/test_CustomEvents_Events.getAction_flat__API.getProcessedReport_day.xml

```diff
@@ -404,8 +408,163 @@
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>event action Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long ---&gt; SHOULD APPEAR IN TEST OUTPUT NOT TRUNCATED &lt;---</Events_EventAction>
+				<Events_EventName>event name Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long Extremely long ---&gt; SHOULD APPEAR IN TEST OUTPUT NOT TRUNCATED &lt;---</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play25%</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play25%</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play50%</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play50%</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play75%</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>play75%</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playEnd</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playEnd</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playStart</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playTrailer</Events_EventAction>
+				<Events_EventName>Ponyo (崖の上のポニョ)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playTrailer</Events_EventAction>
+				<Events_EventName>Princess Mononoke (もののけ姫)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>playTrailer</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>Purchase</Events_EventAction>
+			</row>
+			<row>
+				<Events_EventAction>rating</Events_EventAction>
+				<Events_EventName>La fiancée de l'eau</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>rating</Events_EventAction>
+				<Events_EventName>Spirited Away (千と千尋の神隠し)</Events_EventName>
+			</row>
+			<row>
+				<Events_EventAction>Search</Events_EventAction>
+				<Events_EventName>Search query here</Events_EventName>
+			</row>
+		</result>
```

Comment:
I see now there will be rows w/ dimensions even if there is no data. Will this be a BC break? I wonder if the mobile app or other types of apps will experience different behavior because of this.

Reply:
cc @tsteur in case he has some insight